manifests/o/OpenCPN/OpenCPN/5.8.4/OpenCPN.OpenCPN.locale.en-US.yaml

```
@@ -0,0 +1,36 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: OpenCPN.OpenCPN
+PackageVersion: 5.8.4
+PackageLocale: en-US
+Publisher: opencpn.org
+PublisherUrl: https://github.com/OpenCPN
+PublisherSupportUrl: https://github.com/OpenCPN/OpenCPN/issues
+Author: opencpn.org
+PackageName: OpenCPN
+PackageUrl: https://opencpn.org/index.html
+License: GPL-2.0, GPL-3.0, LGPL-3.0, LGPL-2.0
+LicenseUrl: https://github.com/OpenCPN/OpenCPN/blob/Release_5.8.2/LICENSING

```

```suggestion
LicenseUrl: https://github.com/OpenCPN/OpenCPN/blob/master/LICENSING
```
Use master branch URL so that it can be carried forward in update PRs